ckanext/ontario_theme/templates/scheming/form_snippets/help_text.html

```diff
@@ -0,0 +1,8 @@
+{% import 'macros/form.html' as form %}
+
+{%- if field.help_text -%}
+    {{- form.info(
+        text=h.render_markdown(h.scheming_language_text(field.help_text), True, True),
```

Comment:
For future, could call the `render_markdown()` helper with named args like this for readability.
`h.render_markdown(h.scheming_language_text(field.help_text), allow_html=True)`